bool forward
bool backward
int32 pwm
